app/models/bulkrax/importer.rb

```diff
@@ -60,17 +60,21 @@ def record_status
     def mapping
       @mapping ||= if self.field_mapping.blank? || self.field_mapping == [{}]
                      if parser.import_fields.present? || self.field_mapping == [{}]
-                       ActiveSupport::HashWithIndifferentAccess.new(
-                         parser.import_fields.reject(&:nil?).map do |m|
-                           Bulkrax.default_field_mapping.call(m)
-                         end.inject(:merge)
-                       )
+                      default_field_mapping
                      end
                    else
-                     self.field_mapping
+                     self.field_mapping.merge(default_field_mapping)
```

Comment:
```suggestion
                     default_field_mapping.merge(self.field_mapping)
```

Reply:
I think the reverse is needed so that the client's mappings override the defaults.